### PR TITLE
Docs: Use third-person singular verbs in docblocks. (2 of 2)

### PR DESCRIPTION
**Pull request by costdev**
Trac ticket: https://core.trac.wordpress.org/ticket/57840

### CODE REVIEW

File: src/wp-admin/includes/class-wp-posts-list-table.php [hunk 1]
```diff
@@ -917,8 +917,8 @@ private function _display_rows_hierarchical( $pages, $pagenum = 1, $per_page = 2
 	}
 
 	/**
-	 * Given a top level page ID, display the nested hierarchy of sub-pages
-	 * together with paging support
+	 * Displays the nested hierarchy of sub-pages
```
Comment (audrasjb) [inline src/wp-admin/includes/class-wp-posts-list-table.php]: Suggestion:
```
	 * Displays the nested hierarchy of sub-pages together with paging
	 * support, based on a top level page ID.
```

File: src/wp-admin/includes/class-wp-privacy-requests-table.php [hunk 2]
```diff
@@ -527,7 +527,7 @@ public function column_email( $item ) {
 	}
 
 	/**
-	 * Next steps column. Overridden by children.
+	 * Returns the markup for the Next steps column. Overridden by children.
```
Comment (audrasjb) [inline src/wp-admin/includes/class-wp-privacy-requests-table.php]: Remove capital N on "Next"

**Review by audrasjb — CHANGES_REQUESTED** [review]
Thanks @costdev I found two small issues :)

File: src/wp-admin/includes/class-wp-posts-list-table.php (hunk 1 above)
Comment (costdev) [reply src/wp-admin/includes/class-wp-posts-list-table.php]: Fixed in [e3f2e97](https://github.com/WordPress/wordpress-develop/pull/4552/commits/e3f2e9783fd2a232f4275a6abeea815c1b133a07).

File: src/wp-admin/includes/class-wp-privacy-requests-table.php (hunk 2 above)
Comment (costdev) [reply src/wp-admin/includes/class-wp-privacy-requests-table.php]: Fixed in [e3f2e97](https://github.com/WordPress/wordpress-develop/pull/4552/commits/e3f2e9783fd2a232f4275a6abeea815c1b133a07).

**Comment by audrasjb** [comment]
Thanks for the update! 
Committed in https://core.trac.wordpress.org/changeset/55911

**Pull request closed without merging**